test/integration/current_category_test.rb

```diff
@@ -7,9 +7,9 @@ class CurrentCategoryTest < ActionDispatch::IntegrationTest
   setup do
     travel_to Time.zone.parse("2000-01-01 00:00:00")
 
-    site = Site.create!(name: "daimon-news", fqdn: "www.example.com")
+    @site = Site.create!(name: "daimon-news", fqdn: "www.example.com")
```

Comment:
This shouldn't depend default factory data. Because it is so fragile.
If you want to assert `Site#category_title_format`, you can specify it in here.

----

そもそも `Site#category_title_format` のアサートを他のテストを一緒くたにしないほうがいいのでは…？